.github/workflows/CI.yml

```diff
@@ -77,6 +77,10 @@ jobs:
       - name: build
         run: cargo build --manifest-path ${{ matrix.conf.name }}/Cargo.toml --features "${{ matrix.conf.features }}"
         if: matrix.rust != 'nightly'
+      - name: Test ${{ matrix.conf.name }}/sys
+        run: cargo test
+        working-directory: ${{ matrix.conf.name }}/sys
+        if: matrix.conf.name == 'graphene'
```

Comment:
You just need to change the value in https://github.com/gtk-rs/gtk-rs-core/pull/73/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R26

Reply:
Parameter `test_sys` does nothing. It is not used anywhere in the `CI` action.

Reply:
@bilelmoussaoui What should we do about this?

Reply:
wait what. It definitely used to do something, any ideas @GuillaumeGomez ?

Reply:
Please copy it from https://github.com/gtk-rs/gtk4-rs/blob/master/.github/workflows/CI.yml#L37-L39 for now :-)